### DISCUSSION TITLE
envkey-rs rust sdk

### DISCUSSION THREAD

**Discussion by dezren39** [envkey/envkey#78] [Ideas and feedback]
hello, i am writing you in support of the development of a rust sdk for envkey. in the mean time i can use the command line binary. thank you,

**Comment by danenania** [comment]
Thanks for the post. We're planning to add a Rust sdk within the next couple of months. Hopefully [envkey-source](https://docs-v2.envkey.com/docs/envkey-source) will work well for you in the meantime.
